packages/demo-wallet-client/src/ducks/sep31Send.ts

```diff
@@ -324,15 +324,20 @@ export const submitSep31SendTransactionAction = createAsyncThunk<
       });
 
       // Poll transaction until ready
-      await pollTransactionUntilReady({
+      const getSep31Tx = await pollTransactionUntilReady({
         sendServer,
         transactionId: postResponse.transactionId,
         token,
       });
 
+      const amountIn = getSep31Tx?.transaction?.amount_in;
+      if (amountIn === undefined) {
+        throw new Error(`"amount_in" is missing from the GET /transaction response`);
+      }
+
       // Send payment
       await sendPayment({
-        amount: amount.amount,
+        amount: getSep31Tx!.transaction!.amount_in as string,
```

Comment:
We could use `amountIn` here.

Reply:
https://github.com/stellar/stellar-demo-wallet/pull/275/commits/5bf0e1caac9386e27bf1fa028723fd0aa14e52f1